source-code/sdk-js/src/helpers/ast.ts

```diff
@@ -0,0 +1,108 @@
+import type { CallExpression, Node, Program } from "estree"
+import { walk } from "estree-walker"
+import { get } from "lodash"
+
+type wrapVariableDeclaration = (
```

Comment:
Types should always begin with an uppercased letter

Reply:
Will fix